### DISCUSSION TITLE
Anyone has subdomains for fortnite service api?

### DISCUSSION THREAD

**Discussion by Damnbuoy** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#336] [Q&A] [answered]
I saw that account service had multiple subdomains. Im trying to access the api for old fortnite version.
These are domains im aware of:

```
https://fngw-mcp-gc-livefn.ol.epicgames.com/fortnite/api/version
https://fortnite-public-service-prod11.ol.epicgames.com/fortnite/api/version
```

Please let me know if u have any except these.

**Comment by Krowe-moh** [comment, marked as answer]
none exist for old versions

these are different environments (prod,live,dev,local and ect)
